software/riscv-tools/Makefile

```diff
@@ -28,7 +28,7 @@
 .DEFAULT_GOAL := help
 
 TARGET_ARCH       := rv32imaf
-TARGET_ABI        := ilp32 # TODO: Test ilp32f for potential performance improvement
+TARGET_ABI        := ilp32f # TODO: Test ilp32f for potential performance improvement
```

Comment:
Seems like this todo might be unnecessary. Can you file an issue for it, if it still is?

Reply:
Forgot to remove it.